tests/responses/update_customer.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
create_customer and update_customer have the exact same content and response (namely a single full customer record) Don't use 2 response mock files for that..